src/collision/cm_trace.h

```diff
@@ -24,36 +24,42 @@
 #include "cm_types.h"
 
 cm_trace_t Cm_BoxTrace(const vec3_t start, const vec3_t end, const vec3_t mins, const vec3_t maxs,
-                       const int32_t head_node, const int32_t contents);
+                       const int32_t head_node, const int32_t contents, const mat4_t *matrix, const mat4_t *inverse_matrix);
 
-/**
- * @brief Collision detection for non-world models. Rotates the specified end
- * points into the model's space, and traces down the relevant subset of the
- * BSP tree. For inline BSP models, the head node is the root of the model's
- * subtree. For mesh models, a special reserved box hull and head node are
- * used.
- *
- * @param start The trace start point, in world space.
- * @param end The trace end point, in world space.
- * @param mins The bounding box mins, in model space.
- * @param maxs The bounding box maxs, in model space.
- * @param head_node The BSP head node to recurse down.
- * @param contents The contents mask to clip to.
- * @param matrix The matrix of the entity to clip to.
- * @param inverse_matrix The inverse matrix of the entity to clip to.
- *
- * @return The trace.
- */
-cm_trace_t Cm_TransformedBoxTrace(const vec3_t start, const vec3_t end,
-								  const vec3_t mins, const vec3_t maxs,
-								  const int32_t head_node, const int32_t contents,
-                                  const mat4_t matrix, const mat4_t inverse_matrix);
-
-void Cm_EntityBounds(const solid_t solid, const vec3_t origin, const vec3_t angles,
+void Cm_EntityBounds(const solid_t solid, const vec3_t origin, const vec3_t angles, const mat4_t matrix,
```

Comment:
I'd like to retain whatever is still relevant from the doxygen comments in this file. There's some good foundational info in there for anyone new to the code.